certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -193,6 +198,8 @@ def choose_vhost(self, target_name):
         :rtype: :class:`~certbot_nginx.obj.VirtualHost`
 
         """
+        if self.vhost is not None:
```

Comment:
This was supposed to get set to vhost at the end of this function. Instead, I set it to None.